code/go/0chain.net/smartcontract/dbs/event/users.go

```diff
@@ -0,0 +1,87 @@
+package event
+
+import (
+	"errors"
+	"fmt"
+
+	"0chain.net/smartcontract/dbs"
+	"gorm.io/gorm"
+)
+
+type Users struct {
```

Comment:
We should keep entity definition singular. It will be automatically translate into plural when converted to table_name (i.e user).